client/reader/user-profile/views/test/lists.tsx

```diff
@@ -0,0 +1,145 @@
+/**
+ * @jest-environment jsdom
+ */
+
+import { render, screen } from '@testing-library/react';
+import React from 'react';
+import { UserLists } from '../lists';
+
+// Mock the EmptyContent component
+jest.mock( 'calypso/components/empty-content', () => ( {
+	__esModule: true,
+	default: ( { icon, line } ) => (
+		<div data-testid="empty-content">
+			{ icon && <div data-testid="empty-content-icon">{ icon }</div> }
+			{ line && <p data-testid="empty-content-line">{ line }</p> }
+		</div>
+	),
+} ) );
+
+describe( 'UserLists', () => {
+	const defaultUser = {
```

Comment:
```suggestion
	const defaultUser: UserData = {
```

Reply:
Done in 8d1d8657331510074499b42bd75039b114df2937